### ISSUE TITLE
Github for Ideas

### ISSUE THREAD

**Issue by o-jasper** [demirb/make-my-idea#18]
([This one](https://github.com/demirb/make-my-idea#github-for-ideas)) Basically a wiki, discussion-forum, and a list of supporters?

**Comment by demiculus** [comment]
Yep, you can say visualize it better if I say a reversed kickstarter or product hunt for ideas. 

Reversed kickstarter: 
In kickstarter creators create a page, want people to buy it. 
Reversed kickstarter people want something to buy, then creators create it.
